Essentials/src/com/earth2me/essentials/utils/NumberUtil.java

```diff
@@ -105,4 +105,11 @@ public static boolean isInt(final String sInt) {
         }
         return true;
     }
+
+    public static boolean isPosInt(final String sInt) {
```

Comment:
```suggestion
    public static boolean isPositiveInt(final String sInt) {
```

As opposed to "positional" or something similar, which is what I thought this was at first glance